src/layouts/Layout.jsx

```diff
@@ -4,13 +4,18 @@ import React from 'react';
 import { useSession } from '@hooks';
 // Material UI Imports
 import Box from '@mui/material/Box';
+// Custom hook imports
+import useNotification from '../hooks/useNotification';
 // Component Imports
 import { NavBar } from '../components/NavBar';
 import { InactivityMessage } from '../components/Notification';
 import Footer from '../components/Footer/Footer';
+import NotificationContainer from '../components/Notification/NotificationContainer';
 
 const Layout = ({ ariaLabel, children }) => {
   const { session } = useSession();
+  const notify = useNotification();
```

Comment:
You could directly call the `state` from the `useNotificiation` hook with destructuring:

`const { state } = useNotification()`